scripts/nyc-council-councilmatic-staging-crontasks

```diff
@@ -1,2 +1,3 @@
 # /etc/cron.d/nyc-council-councilmatic-staging-crontasks
 0 0 * * * datamade cd /home/datamade/nyc-council-councilmatic && /home/datamade/.virtualenvs/nyc-council-councilmatic/bin/python manage.py import_data >> /tmp/nyc-council-councilmatic-loaddata.log 2>&1
+0 0 * * * datamade cd /home/datamade/nyc-council-councilmatic && /home/datamade/.virtualenvs/nyc-council-councilmatic/bin/python manage.py convert_rtf >> /tmp/convert_rtf.log 2>&1
```

Comment:
What's the reason for doing this in one command on the live site, but two concurrent commands on staging (if I'm interpreting this cron correctly)?

Reply:
We are chaining the commands (and also using flock) to prevent overlap between `import_data` and `convert_rtf`, which runs every 15 minutes on the production server.

The staging server runs these commands once per day - the likelihood of overlap seems minimal. So, for readability, I did not chain. 

With that said, can we imagine a scenario in which NYC might update thousands of bills in one day, i.e., the conversion script would need more than 24 hours? Maybe yes. Then, we should 
 implement "flock" on the staging server, too. Thoughts @hancush?

Reply:
Ahhhh, so I was misinterpreting! I was slow to understand we want to prevent one `convert_rtf` command from conflicting with another `convert_rtf` command, rather preventing conflict between `import_data` and `convert_rtf`. This is possibly because the chaining suggested to me that `import_data` needed to finish before `convert_rtf` could run, which doesn't seem to be the case... is that right?

@fgregg – For learning purposes, what makes chaining the commands in production safer than having two separate commands fire? Is it that we don't want / need `convert_rtf` to run if `import_data` fails?

Reply:
@hancush - we want `import_data` to finish before running the conversion. We also do not want the conversion script to be running while import another import begins. So...one script-at-a-time.

Reply:
In that case, I think it makes sense to chain the commands on the staging site (won't they _always_ be running at the same time if we start them both at midnight?), but I don't think we need to worry about flock.

Reply:
Okay - I updated the staging cron (without flock!) but with chained commands.